.travis.yml

```diff
@@ -6,10 +6,10 @@ matrix:
       compiler: gcc
     - os: linux
       compiler: clang
-#    - os: osx
-#      compiler: gcc
-#    - os: osx
-#      compiler: clang
+    - os: osx
+      compiler: gcc
```

Comment:
FYI, 'gcc' is aliased to clang/llvm on OS X workers, if you really want gcc, you need to specify gcc-4.8 for instance (I don't remember if it is installed by default, but you can still install it through `brew`)